lib/ansible/cli/doc.py

```diff
@@ -436,6 +461,10 @@ def init_parser(self):
         exclusive.add_argument("--metadata-dump", action="store_true", default=False, dest='dump',
                                help='**For internal use only** Dump json metadata for all entries, ignores other options.')
 
+        self.parser.add_argument("--dont-fail-on-errors", action="store_true", default=False, dest='dont_fail_on_errors',
```

Comment:
```suggestion
        self.parser.add_argument("--no-fail-on-errors", action="store_true", default=False, dest='dont_fail_on_errors',
```
A common pattern in CLI args is yup use "no".